edc-controlplane/edc-controlplane-memory-hashicorp-vault/src/main/docker/Dockerfile

```diff
@@ -26,17 +26,13 @@ HEALTHCHECK NONE
 
 RUN wget ${OTEL_AGENT_LOCATION} -O /tmp/opentelemetry-javaagent.jar
 
-FROM alpine:3.17.1
+FROM eclipse-temurin:11
```

Comment:
Hey, sorry to already reviewing it, while it's still in draft! But please use a tag as specific as possible. Otherwise we will not have reproducible builds, which we need to provide hotfixes for a specific release!